.github/workflows/test.yaml

```diff
@@ -0,0 +1,87 @@
+name: Test
+
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+
+concurrency: ${{ github.workflow }}-${{ github.ref }}
+
+jobs:
+  test-skaffold:
+    name: Skaffold
+    runs-on: ubuntu-latest
+    steps:
+      - name: Setup Skaffold
+        run: curl -Lo skaffold https://storage.googleapis.com/skaffold/releases/latest/skaffold-linux-amd64 && sudo install skaffold /usr/local/bin/
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: 18
+
+      - name: Checkout Repo
+        uses: actions/checkout@v2
+
+      - name: Start Minikube
+        run: minikube start --uuid 00000000-0000-0000-0000-000000000001 --extra-config=apiserver.enable-admission-plugins=ValidatingAdmissionWebhook
```

Comment:
Is minikube installed on ubuntu by default?

Reply:
It's not Ubuntu by default but GH Actions hosted runners which has a lot of stuff preinstalled. You can see here - https://github.com/actions/runner-images/blob/main/images/linux/Ubuntu2204-Readme.md.

Reply:
I'm wondering whether we should test agains multiple Kubernetes version. As this is not requesting a specific one, it will vary between minikube releases.

We can run the test in parallel for like the last 3-5 kubernetes version and see the results, as most Kubernetes add-ons in the CNCF space do.
Whatchu think @WitoDelnat?

Reply:
Somehow missed this thread and put my answer as separate comment - https://github.com/kubeshop/monokle-admission-controller/pull/8#issuecomment-1770716430.